tests/python/tank_test/tank_test_base.py

```diff
@@ -157,6 +158,12 @@ def setUpModule():
     # copy tank engine code into place
     os.makedirs(os.path.join(install_dir, "engines"))
 
+def tearDownModule():
+    # Last chance to delete what was created in `TANK_TEMP`
```

Comment:
- What do we expect to is being deleted here that isn't already handled? Please elaborate in comments.
- How does this work with `_move_data`? The idea with move data was that we would keep one version of the fixtures around so that in case something fails you can easily inspect the contents of the fixtures. But because we are only keeping one copy, that shouldn't grow over time. I think we still want that behaviour - keeping one backup is fine, it's the fact that our temp data grows over time that would be great if we could fix.